code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -4,14 +4,12 @@
 	damage = 15
 	damage_type = TOX
 	nodamage = 0
-	stamina = 50
-	eyeblur = 10
-	knockdown = 10
-	slur = 5
+	stamina = 25
 
 /obj/item/projectile/energy/bolt/halloween
 	name = "candy corn"
 	icon_state = "candy_corn"
 
 /obj/item/projectile/energy/bolt/large
-	damage = 40
+	damage = 25
+	stamina = 20

```

Comment:
You'll be in crit faster than getting stamstunned, is that intended?

Reply:
I don’t think that’ll happen? Isn’t it the combined stam and damage? I do want the large ebow to be more lethal-the station doesn’t need more stuns- but maybe I’ll make it straight lethal or something.

Reply:
Stam stun will happen first yes

Reply:
oh right it's based on hp not maxhp

75 hp-20 stam
50-<40
**25-<60**